r/configure.win

```diff
@@ -44,7 +44,7 @@ else
   RWINLIB="../windows/$(ls windows/ | grep ^arrow- | tail -n 1)"
 fi
 OPENSSL_LIBS="-lcrypto -lcrypt32"
-MIMALLOC_LIBS="-lbcrypt -lpsapi"
+MIMALLOC_LIBS="-lbcrypt -lpsapi -lole32"
```

Comment:
Should this be somewhere else?

Reply:
No, I didn't see your earlier comment so I didn't understand why you were adding this.  I get it now, sorry.

Reply:
Ah, sorry, thought this was on the other PR where I was questioning this.  I will let someone with more knowledge of the build process chime in on whether this is correct or not.  I thought the TZ stuff didn't support Windows anyways?

Reply:
No worries, thanks for responding :)
The TZ functionality of the vendored library (date.h) needs to download a timezone DB on windows to work. There's [a jira to set that up](https://issues.apache.org/jira/browse/ARROW-13168) so this will eventually need to compile on windows.